client/signup/steps/user/index.jsx

```diff
@@ -444,7 +444,7 @@ export class UserStep extends Component {
 		if ( isWooOAuth2Client( oauth2Client ) && wccomFrom ) {
 			isSocialSignupEnabled = true;
 		}
-
+		// debugger;
```

Comment:
👋🏼

Reply:
I knew I would forget it when I commented it out...and I did 😭  Fixed in ead3ba3f86eda21866d699d1cac9abd6b9acf70c.